ReactWindows/ReactNative.Shared/Views/View/ReactViewControl.cs

```diff
@@ -1,24 +1,37 @@
-﻿using System;
+using System;
 
 #if WINDOWS_UWP
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Data;
 #else
 using System.Windows;
 using System.Windows.Controls;
 #endif
 
-namespace ReactNative.UIManager
+namespace ReactNative.Views.View
 {
     /// <summary>
-    /// Represents a Canvas with an optional Border inside.
+    /// Native control for React view.
     /// </summary>
-    public class BorderedCanvas : Canvas
+    public class ReactViewControl : UserControl
     {
+        private readonly Canvas _canvas;
+
         private Border _border = null;
 
         /// <summary>
+        /// Instantiates the <see cref="ReactViewControl"/>. 
+        /// </summary>
+        public ReactViewControl()
+        {
+            Content = _canvas = new Canvas
+            {
+                HorizontalAlignment = HorizontalAlignment.Stretch,
+                VerticalAlignment = VerticalAlignment.Stretch,
```

Comment:
Why do we need to set this. It wasn't being set before was it?

Reply:
If these values aren't set, the Canvas `Width`/`Height` will never be set. We could add property bindings similar to @antonkh's approach for the lazy Border, but these values also guarantee that the Canvas fills the parent user control.